src/components/RepositoryGoals.js

```diff
@@ -45,6 +47,8 @@ function RepositoryGoals({user}) {
     dispatch({type: "UPDATE", payload: updatedRepos});
   };
 
+  const data = repository.data && JSON.parse(repository.data.text)
```

Comment:
**Suggestion** :heavy_plus_sign: :heavy_minus_sign: 

Based on https://github.com/open-sauced/open-sauced/pull/616/files#r439953193

```suggestion
  const data = repository.data && JSON.parse(repository.data.text);
```

Reply:
This will be caught in the #657